docs/UserGuide.md

```diff
@@ -1736,8 +1736,47 @@ Format: `deleteconsult CONSULTATION_INDEX`
 
 ## FAQ
 
-**Q**: How do I transfer my data to another computer?<br>
-**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous F.A.K.E.J.A.R.V.I.S. home folder.
+<box type="info" seamless icon=":fas-question:">
+
+<md>
+
+**QNS**: How do I transfer my data to another computer?<br>
+
+**ANS**: You can install the app in the other computer and replace the sample data files with the data files in your previous F.A.K.E.J.A.R.V.I.S. folder.
```

Comment:
I'm not sure if "QNS" and "ANS" are fit for use here as readers may be easily confused between the two as to how they look similar.

Reply:
ok, will change it back to "Q" and "A"